### ISSUE TITLE
IE any version issue

### ISSUE THREAD

**Issue by usualdesigner** [select2/select2#3731]
If 'multiple' option is true and 'placeholder' option is not null - dropdown is not closing after selecting first option. It closing only after select second option. It is reproducing in any version of the IE.

**Comment by kevin-brown** [comment]
This is a duplicate of https://github.com/select2/select2/issues/3300 and will be fixed in 4.0.1.
